docs/operations/security-overview.md

```diff
@@ -0,0 +1,244 @@
+
+Druid uses Jetty as its embedded web server. Therefore you refer to [Understanding Certificates and Keys](https://www.eclipse.org/jetty/documentation/current/configuring-ssl.html) for complete instructions. 
+
+
+## Enable an authenticator
+
+To authenticate requests in Druid, you configure an Authenticator. Authenticator extensions exist for HTTP basic authentication, LDAP, and Kerberos.  
+
+The following takes you through sample configuration steps for enabling basic auth:  
+
+1. Add the `basic-auth` extension to `druid.extensions.loadList` in common.runtime.properties. For the quickstart installation, for example, the properties file is at `conf/druid/cluster/_common`:
+   ```
+   druid.extensions.loadList=["druid-basic-security", "druid-histogram", "druid-datasketches", "druid-kafka-indexing-service", "imply-utility-belt"]
+   ```
+2. Configure the basic Authenticator, Authorizer, and Escalator settings in the same common.runtime.properties file. For example:
+   ```
+   # Druid basic security
+   druid.auth.authenticatorChain=["MyBasicMetadataAuthenticator"]
+
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.type=basic
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.initialAdminPassword=password1
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.initialInternalClientPassword=password2
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.credentialsValidator.type=metadata
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.skipOnFailure=false
+   druid.auth.authenticator.MyBasicMetadataAuthenticator.authorizerName=MyBasicMetadataAuthorizer
+
+   # Escalator
+   druid.escalator.type=basic
+   druid.escalator.internalClientUsername=druid_system
+   druid.escalator.internalClientPassword=password2
+   druid.escalator.authorizerName=MyBasicMetadataAuthorizer
+
+   druid.auth.authorizers=["MyBasicMetadataAuthorizer"]
+
+   druid.auth.authorizer.MyBasicMetadataAuthorizer.type=basic
+   ```
+
+3. Restart the cluster. 
+
+See [Basic Security](../development/extensions-core/druid-basic-security) for more information. For more on authentication extensions, see [Kerberos](../development/extensions-core/druid-kerberos), [Authentication and Authorization](../design/auth), and [Authentication and Authorization](../design/auth) 
+
+
+## Enable authorizors
+
+After enabling the basic auth extension, you can add users, roles, and permissions via the Druid Coordinator `user` endpoint. Note that you cannot assign permissions directly to individual users. They must be assigned through roles. 
+
+The following diagram depicts the authorization model, and the relationship between users, roles, permissions, and resources.
+ 
+![Druid Security model](../assets/security-model-2.png "Druid security model") 
+
+
+The following steps walk through a sample setup procedure:  
+
+> The Coordinator API port is 8081 for non-TLS connections and 8281 for secured connections.
```

Comment:
```suggestion
> The default Coordinator API port is 8081 for non-TLS connections and 8281 for secured connections.
```

Reply:
Would also suggest mentioning that if TLS is enabled, the user may have to adjust the example curl commands to account for that (e.g., they used a self-signed certificate for the Druid servers that's not trusted by curl by default)